src/Microsoft.ML.Data/Transforms/KeyToVector.cs

```diff
@@ -723,15 +723,11 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColInfo info, string src
     }
 
     /// <summary>
-    /// Estimator for <see cref="KeyToVectorMappingTransformer"/>. Converts the key types back to their original vectors.
-    /// </summary>
-    /// <summary>
-    /// Utilizes KeyValues <see cref="AnnotationInfo"/> of the input column, to map keys to a vector representing the original value.
-    /// Maps zero values of the <see cref="KeyDataViewType"/> are mapped to the <see langword="default"/> value of the output type.
+    /// Estimator for <see cref="KeyToVectorMappingTransformer"/>. Maps the value of a key
+    /// into a floating point vector representing the value.
```

Comment:
>floating point [](start = 15, length = 14)

Should we avoid float in the descriptions? maybe: known-sized vector of <see cref="System.Single"/>